src/amber/controller/helpers/render.cr

```diff
@@ -37,5 +37,30 @@ module Amber::Controller
         Kilt.render("#{{{path}}}/{{filename.id}}")
       {% end %}
     end
+
+    protected def respond_with(html : String? = nil, json : Hash | String? = nil, xml : String? = nil, text : String? = nil)
```

Comment:
Can we remove this method?`responders.cr` does the format very nicely.

Reply:
Yeah. Weird that shouldn't be there.

Reply:
Done!